### PR TITLE
add BSP for VCC-GND YD-RP2040

### PR DESCRIPTION
**Pull request by nleguen**
This just add support for the VCC-GND Studio YD-RP2040, a cheap Chinese RP2040 board I purchased.

### CODE REVIEW

File: boards/vcc-gnd-yd-rp2040/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# [vcc-gnd-yd-rp2040] - Board Support for the [VCC-GND Studio YD-RP2040]
+
+You should include this crate if you are writing code that you want to run on
+a [VCC-GND Studio YD-RP2040] - a PCB for the RP2040 chip with USB-C port, WS1812 RGB LED on GPIO23, user key on GPIO24 and built-in blue LED.
```
Comment (9names) [inline boards/vcc-gnd-yd-rp2040/README.md:4]:
```suggestion
a [VCC-GND Studio YD-RP2040] - a PCB for the RP2040 chip with USB-C port, WS2812 RGB LED on GPIO23, user key on GPIO24 and built-in blue LED.
```

**Review by 9names — CHANGES_REQUESTED** [review]
Please remove the pinout image from this PR: it is 1.35MB.
Provide a link to the original site for this information instead.

**Comment by nleguen** [comment]
I removed the png file and changed the link to the board info. I also fixed the typo.

**Comment by 9names** [comment]
Thanks. Could you also add a link to the board from main/README.md?
Use the other board entries as a template for this.

**Comment by nleguen** [comment]
done

**Review by 9names — APPROVED** [review]
LGTM, thanks!

**Pull request merged**